monitoring/tracing/errors.go

```diff
@@ -2,17 +2,16 @@
 package tracing
 
 import (
-	"go.opencensus.io/trace"
+	prysmTrace "github.com/prysmaticlabs/prysm/v5/monitoring/tracing/trace"
+	"go.opentelemetry.io/otel/codes"
+	"go.opentelemetry.io/otel/trace"
 )
 
 // AnnotateError on span. This should be used any time a particular span experiences an error.
-func AnnotateError(span *trace.Span, err error) {
+func AnnotateError(span trace.Span, err error) {
 	if err == nil {
 		return
 	}
-	span.AddAttributes(trace.BoolAttribute("error", true))
-	span.SetStatus(trace.Status{
-		Code:    trace.StatusCodeUnknown,
-		Message: err.Error(),
-	})
+	span.SetAttributes(prysmTrace.BoolAttribute("error", true))
```

Comment:
```suggestion
	span.RecordError(err)
```
I think that's a better way to set the errors. See https://github.com/open-telemetry/opentelemetry-go/blob/97ee172f1f9af2abb8db7f17567ecdf23779103d/sdk/trace/span.go#L452